packages/css/src/checkbox/checkbox.scss

```
@@ -0,0 +1,145 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+.amsterdam-checkbox__input {
+  appearance: none;
+  margin-block: 0;
+  margin-inline: 0;
+  opacity: 0%; /* to hide native input field in older iOS */
+
+  &:focus-visible + label {
+    outline: auto;
+    outline-offset: var(--amsterdam-checkbox-outline-offset);
+  }
+}
+
+.amsterdam-checkbox__label {
+  color: var(--amsterdam-checkbox-color);
+  cursor: pointer;
+  display: inline-flex;
+  font-family: var(--amsterdam-checkbox-font-family);
+  font-size: var(--amsterdam-checkbox-narrow-font-size);
+  font-weight: 400;
+  gap: 0.5rem;
+  line-height: var(--amsterdam-checkbox-line-height);
+  padding-block: 0.5rem;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-checkbox-wide-font-size);
+  }
+}
+
+.amsterdam-checkbox__input:disabled + .amsterdam-checkbox__label {
+  color: var(--amsterdam-checkbox-disabled-color);
+  cursor: not-allowed;
+}
+
+.amsterdam-checkbox__input:focus-visible + .amsterdam-checkbox__label,
+:not(.amsterdam-checkbox__input:disabled) + .amsterdam-checkbox__label:hover {
+  color: var(--amsterdam-checkbox-hover-color);
+  text-decoration: underline;
+  text-underline-offset: 3px;
+}
+
+.amsterdam-checkbox__label__checkmark {
+  align-items: center;
+  display: flex;
+  flex-shrink: 0;
+  height: calc(var(--amsterdam-checkbox-checkmark-narrow-size) * var(--amsterdam-checkbox-checkmark-multiplier));
+  width: 1.5rem;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    height: calc(var(--amsterdam-checkbox-checkmark-wide-size) * var(--amsterdam-checkbox-checkmark-multiplier));
+  }
+}
+
+.amsterdam-checkbox__label__checkmark::after {
+  box-shadow: var(--amsterdam-checkbox-checkmark-box-shadow);
+  content: "";
+  height: 1.5rem;
+  width: 100%;
+}
+
+// Checkmark checked state
+.amsterdam-checkbox__input:checked + .amsterdam-checkbox__label .amsterdam-checkbox__label__checkmark::after {
```

Parent selector not really needed unless something is conflicting.

```suggestion
.amsterdam-checkbox__input:checked + .amsterdam-checkbox__label__checkmark::after {
```